lib/remote_service.dart

```diff
@@ -90,16 +87,23 @@ class RemoteService {
           detail['location']['postcode'],
           detail['location']['region']);
 
+      var photoListed = detail['photos'] as List;
```

Comment:
call the var `photoList`, also remove `var`, we can change it to `final List<Photo>` as this value will not change

Reply:
done for now